clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/main/commands/ConnectCommand.as

```diff
@@ -1,5 +1,5 @@
 package org.bigbluebutton.lib.main.commands {
-	
```

Comment:
The FlexFormatter definition includes tabs on the empty lines, but the edit to this file removes them all.